pennylane/beta/plugins/default_tensor.py

```diff
@@ -298,7 +339,7 @@ def expval(self, observable, wires, par):
             offset = len(w)
             tensors.append(self._reshape(A, [2] * offset * 2))
 
-        nodes = self.create_nodes_from_tensors(tensors, wires, observable, key="observables")
+        nodes = self._create_nodes_from_tensors(tensors, wires, observable, key="observables")
```

Comment:
Whats the reasoning behind making this private? Just curious.

Reply:
it's just a utility method used by the class itself